schemas/data/poi-entries.schema.json

```diff
@@ -42,6 +42,9 @@
   "allOf": [
     {
       "$ref": "#/definitions/metric"
```

Comment:
This looks like an error too, it should be `"$ref": "#/definitions/metricdefinition"`